extensions/http-curl/tests/C2FailedUpdateTest.cpp

```diff
@@ -201,13 +199,11 @@ int main(int argc, char **argv) {
 
   controller->load();
   controller->start();
-  waitToVerifyProcessor();
 
+  const bool test_success = verifyLogLinePresenceInPollTime(std::chrono::seconds(10), "Invalid configuration payload", "update failed.");
```

Comment:
Why the extra bool and why don't we do it in the proper overridden function?

Reply:
No longer applicable.